src/event-schemas/resource-event.json

```diff
@@ -1,88 +1,106 @@
 {
-    "$id": "com.amazon.rum.performance_resource_event",
+    "$id": "com.amazon.rum.performance_resource_timing_event",
     "$schema": "https://json-schema.org/draft/2020-12/schema",
     "title": "ResourceEvent",
     "type": "object",
+    "definitions": {
+        "performanceServerTimingPolyfill": {
+            "$id": "#performanceServerTimingPolyfill",
+            "type": "object",
+            "properties": {
+                "description": {
+                    "type": "string"
+                },
+                "duration": {
+                    "type": "number"
+                },
+                "name": {
+                    "type": "string"
+                }
+            },
+            "additionalProperties": false
+        }
+    },
     "properties": {
         "version": {
```

Comment:
Version is not needed is useful for readability. And the old ResourceEvent used version=1.0.0